docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/recommended-tracking-design-web-mobile/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Recommended Tracking Design for Web & Mobile Events"
+date: "2023-10-12"
+sidebar_position: 200
+---
+
+# Recommended Tracking Design for Web & Mobile Events
+
+Depending on your business needs you may decide to come up with your own custom contexts / self-describing events. However, for out-of-the-box solutions Snowplow designed a set of contexts specifically useful for web and mobile tracking which we used as a base for our data models, too. 
```

Comment:
`context entities` here and elsewhere in the intro

Reply:
I think we've actually moved to ideally just calling them entities everywhere (https://deploy-preview-642--snowplow-docs.netlify.app/docs/understanding-your-pipeline/entities/)